src/test/kotlin/calculator/StringAddCalculatorTest.kt

```diff
@@ -0,0 +1,34 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculatorTest {
+    private val calculator = StringAddCalculator
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    fun `빈 문자열 또는 null을 입력할 경우 0을 반환해야 한다`(input: String?) {
+        assertThat(calculator.add(input)).isZero
```

Comment:
object를 아래와 같이 직접호출하시지 않은 이유가 있을까요?

```suggestion
        assertThat(StringAddCalculator.add(input)).isZero
```

Reply:
매번 StringAddCalculator 이라고 치기 불편해서.. 최소한의 의미전달 + 편의성을 위해서 calculator로 쓰게 되었습니다.